hack/deployer/runner/gke.go

```diff
@@ -220,6 +224,34 @@ func (d *GkeDriver) GetCredentials() error {
 	return NewCommand(cmd).AsTemplate(d.ctx).Run()
 }
 
+// patchStorageClass based on standard storageclass, creates new default with "volumeBindingMode: WaitForFirstConsumer"
+func (d *GkeDriver) patchStorageClass() error {
+	log.Println("Patching storage class...")
+
+	if exists, err := NewCommand("kubectl get sc").OutputContainsAny("standard-customized"); err != nil {
+		return err
+	} else if exists {
+		return nil
+	}
+
+	sc, err := NewCommand("kubectl get sc standard -o yaml").Output()
```

Comment:
We should not rely on naming but on the annotation `storageclass.beta.kubernetes.io/is-default-class=true` and `storageclass.kubernetes.io/is-default-class`

Reply:
This was mirroring previous behavior (below). I've changed to use annotations now. https://github.com/elastic/cloud-on-k8s/blob/ee0645187e48e282a9af18425aa846c6fe6655ec/operators/hack/gke-cluster.sh#L91